crowd_anki/config/config_settings.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
+from dataclasses import dataclass
+
+from aqt import mw
+from ..utils.constants import USER_FILES_PATH
+
+@dataclass
+class ConfigSettings():
+    class Properties(Enum):
+        SNAPSHOT_PATH = "snapshot_path"
+        AUTOMATED_SNAPSHOT = "automated_snapshot"
+        SNAPSHOT_ROOT_DECKS = "snapshot_root_decks"
+        EXPORT_DECK_SORT_METHODS = "export_deck_sort_method"
+        EXPORT_DECK_SORT_REVERSED = "export_deck_sort_reversed"
+
+
+    class DeckExportSortMethods(Enum):
+        NO_SORTING = "none"
+        GUID = "guid"
+        FLAG = "flag"
+        TAG = "tag"
+        NOTEMODEL = "notemodel"
+        NOTEMODELID = "notemodelid"
+        FIELD1 = "field1"
+        FIELD2 = "field2"
+
+    snapshot_path: str
+    automated_snapshot: bool
+    snapshot_root_decks: list
+
+    export_deck_sort_reversed: bool
+    export_deck_sort_methods: list
+
+
+    def __init__(self, get_settings = True):
```

Comment:
why do we have this option here? when would we want not to load them?

Reply:
That is for the unit tests, didn't want them to load the values but for the tests to set them manually.

Reply:
so this'll depend on what we'll eventually do with the `config_file` but I think a better option here may be to accept initializer for that dict and if it's none load it from anki, that'd imho proved and more natural/full way to test the thing

Reply:
This can be an improvement for the future, sure.